examples/color.rs

```diff
@@ -0,0 +1,14 @@
+use textplots::{Chart, ColorPlot, PixelColor, Shape};
+
+fn main() {
+    // You can plot several functions on the same chart.
+    // However the resolution of text displays is low, and the result might not be great.
+    println!("\ny = cos(x), y = sin(x) / 2");
+    Chart::new(180, 60, -5.0, 5.0)
+        .linecolorplot(&Shape::Continuous(Box::new(|x| x.cos())), PixelColor::Red)
+        .linecolorplot(
+            &Shape::Continuous(Box::new(|x| x.sin() / 2.0)),
+            PixelColor::Blue,
+        )
+        .display();
+}
```

Comment:
Could we also add a rainbow example like here https://github.com/ftxqxd/drawille-rs/pull/10? 🙏

Reply:
Preferences on if it should be within the same example or a standalone `rainbow` example?

Reply:
Could be in the same file

Reply:
This looks like it's going to be non-trivial or at the least require a good chunk of code to implement. Certainly possible but just don't have time at the moment to add this in

Reply:
@loony-bean  added a rainbows example

Reply:
<img width="689" alt="Screenshot 2021-08-20 at 14 16 31" src="https://user-images.githubusercontent.com/444477/130225984-992b7dff-3893-49ca-a5c3-b258c0833c33.png">

Yikes.. something is not working (macOS Big Sur terminal)

Also take a look at the VSCode terminal output. Looks amazing, but not quiet right

<img width="682" alt="Screenshot 2021-08-20 at 14 26 18" src="https://user-images.githubusercontent.com/444477/130226628-ce196008-fc2c-476d-adce-b8d9bfd7f6e5.png">

Reply:
The MacOS terminal does not support true color and so the results you are seeing are expected

The font used within the VSCode terminal shows all the pips of the braille letters and so that is also expected. 

I suggest trying out iterm2 for a terminal that fully supports color: https://iterm2.com

![Screen Shot 2021-08-20 at 10 57 38 AM](https://user-images.githubusercontent.com/34199479/130260791-9473b58a-45be-482b-9f4a-90873265a256.png)

Reply:
This is looking great! Thank you